editor/db/test/full.pbtxt

```diff
@@ -48,10 +48,10 @@ reactions {
           details: "6"
```

Comment:
volume_includes_solutes was changed to only be present when the measurement is a volume; I changed the measurement a volume so that we could include it in testing.

I think analysis keys were added to make sure that the keys referenced in products were keys in the analyses messages. This is necessary for valid reactions (and maybe to load reactions?) @connorcoley can explain better than I

Reply:
- The field `volume_includes_solutes` should only be defined when the amount is defined by volume, so I thought changing this `moles` amount to a `volume` amount would help illustrate that
- Nathan is right in that the strings referenced in a product's characterization metadata need to correspond to keys in the `analyses` map for the reaction to validate